crypto/src/main/java/net/corda/v5/crypto/KeyUtils.java

```diff
@@ -42,17 +42,16 @@ private KeyUtils() {
      * @param otherKeys An {@link Iterable} sequence of {@link PublicKey}.
      * @return True if <code>key</code> is in otherKeys.
      */
-    public static boolean isKeyInSet(@NotNull PublicKey key, @NotNull Iterable<PublicKey> otherKeys) {
+    public static boolean isKeyInKeys(@NotNull PublicKey key, @NotNull Set<PublicKey> otherKeys) {
```

Comment:
I don't like this name at all, it sounds confusing to me.

Reply:
I agree. I think the original name was actually better. I'd be tempted to call the second argument `set` then `isKeyInSet(key, set)` makes sense.

Reply:
I believe set does not reflect the business side of the second variable. It's too generic.

Reply:
But you can see from the signature that is is a set of `PublicKey`.

But thinking about it now. Does the function provide anything over `set.Contains(key)` where set is a list of `PublicKey`?

Reply:
It also takes care the case where `key` is `CompositeKey`, i.e. checks its leaves then